src/data/whats-new-ids.json

```diff
@@ -59,6 +59,7 @@
   "/whats-new/2021/03/AIOps-made-easy-detect-unusual-changes": "42693",
   "/whats-new/2021/03/nerdlog-weekly-roundup-march-8": "42694",
   "/whats-new/2021/02/nerdlog-weekly-roundup-feb-24": "42695",
+  "/whats-new/2021/03/kubernetes-whats-new": "42696"
```

Comment:
```suggestion
  "/whats-new/2021/03/kubernetes-whats-new": "42696",
```

Reply:
This is probably causing the build to fail here: https://github.com/newrelic/docs-website/pull/1936/checks?check_run_id=2417553534

Reply:
Thanks!